src/SafeModule.js

```diff
@@ -32,6 +32,11 @@ const UNMOCKED_PROPERTY_WHITELIST = {
   removeListeners: true,
 };
 
+const eventEmitterMock = {
+  addListener() {},
+  removeListeners() {},
```

Comment:
just the latter.

see
* https://github.com/facebook/react-native/blob/e774d9991abe29ae5827a854fa64188c7058aa86/Libraries/EventEmitter/NativeEventEmitter.js#L48
* https://github.com/facebook/react-native/blob/e774d9991abe29ae5827a854fa64188c7058aa86/Libraries/EventEmitter/NativeEventEmitter.js#L55